tests/v1/test_plan.py

```diff
@@ -0,0 +1,72 @@
+import pytest
```

Comment:
Write tests to mock the database
Check the current test files for how to go about it

Reply:
@joboy-dev done.